src/main/java/practice/StreamPractice.java

```diff
@@ -1,80 +1,66 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Comparator;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
-    /**
-     * Given list of strings where each element contains 1+ numbers:
-     * input = {"5,30,100", "0,22,7", ...}
-     * return min integer value. One more thing - we're interested in even numbers.
-     * If there is no needed data throw RuntimeException with message
-     * "Can't get min value from list: < Here is our input 'numbers' >"
-     */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(s -> Arrays.asList(s.split(",")))
+                .flatMap(Collection::stream)
+                .map(Integer::parseInt)
+                .filter(i -> i % 2 == 0)
+                .min(Comparator.naturalOrder())
```

Comment:
```suggestion
                .mapToInt(Integer::parseInt)
                .filter(i -> i % 2 == 0)
                .min()
```
We can select MapToInt here and use the .min() method without a comparator